src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -470,13 +472,12 @@ private static bool MatchAttributeTarget(IAttributeTargetSymbol attributeTarget,
                 {
                     //NOTE: ValueText so that we accept targets like "@return", to match dev10 (DevDiv #2591).
                     diagnostics.Add(ErrorCode.WRN_InvalidAttributeLocation,
-                        targetOpt.Identifier.GetLocation(), targetOpt.Identifier.ValueText);
+                        targetOpt.Identifier.GetLocation(), targetOpt.Identifier.ValueText, allowedTargets.ToDisplayString());
```

Comment:
Sorry. A string for what? I don't think attribute targets should be localized here and ```allowedTargets.ToDisplayString()``` doesn't localize them.